src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -500,6 +510,367 @@ function calc_boundary_flux_by_direction_divergence!(surface_flux_values::Abstra
     return nothing
 end
 
+function prolong2mortars!(cache, flux_viscous::Tuple{AbstractArray, AbstractArray},
+                          mesh::TreeMesh{2}, equations_parabolic::AbstractEquationsParabolic,
+                          mortar_l2::LobattoLegendreMortarL2, surface_integral,
+                          dg::DGSEM)
+    flux_viscous_x, flux_viscous_y = flux_viscous                          
+    @threaded for mortar in eachmortar(dg, cache)
```

Comment:
Compared to the implementation for hyperbolic PDEs
https://github.com/trixi-framework/Trixi.jl/blob/d96514f6d58e48b33816f58e02959d167954fdea/src/solvers/dgsem_tree/dg_2d.jl#L800-L803

the viscous fluxes take the role of the variables here.